api/src/org/apache/cloudstack/api/command/user/vm/StopVMCmd.java

```diff
@@ -54,7 +54,7 @@
     private Long id;
 
     @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force stop the VM "
-        + "(vm is marked as Stopped even when command fails to be send to the backend).  The caller knows the VM is stopped.")
+        + "(vm is marked as Stopped even when command fails to be send to the backend, otherwise a force poweroff is attempted).  The caller knows the VM is stopped.")
```

Comment:
Most users (implicitly) assume that the forced parameter will cause a forced stop (hard shutdown or power off). The currently behaviour marks the VM as stopped in the DB when provided (if shutdown operations fails due to host down etc). Backward compatibility is still maintained as during the forced VM stop, VM may still be marked as stopped in case the operations fails to reach the host.